ia/src/protocole.py

```
@@ -31,18 +33,17 @@
     print("Rotation à gauche réussie")
 
 def handle_Take(client, response) -> None:
```

Ruff (0.11.9)

Missing type annotation for function argument `client`

(ANN001)